provider/resources.go

```diff
@@ -493,9 +494,13 @@ var namespaceMap = map[string]string{
 	"aws": "Aws",
 }
 
+var _nslock = sync.Mutex{}
```

Comment:
Curious what happened here, we ran awsMember call from multiple goroutines?

Reply:
That's probably because of all the t.Parallel introduced into the tests? Interesting.